lib/flowViewport/sceneIndex/fvpWireframeSelectionHighlightSceneIndex.cpp

```diff
@@ -474,11 +514,18 @@ WireframeSelectionHighlightSceneIndex::GetChildPrimPaths(const SdfPath &primPath
     SdfPath selectionHighlightMirrorAncestor = _FindSelectionHighlightMirrorAncestor(primPath);
     if (!selectionHighlightMirrorAncestor.IsEmpty()) {
         SdfPath originalAncestor = _GetOriginalPathFromSelectionHighlightMirror(selectionHighlightMirrorAncestor);
-        auto childPaths = GetInputSceneIndex()->GetChildPrimPaths(primPath.ReplacePrefix(selectionHighlightMirrorAncestor, originalAncestor));
-        for (auto& childPath : childPaths) {
-            childPath = childPath.ReplacePrefix(originalAncestor, selectionHighlightMirrorAncestor);
+        auto originalChildPaths = GetInputSceneIndex()->GetChildPrimPaths(primPath.ReplacePrefix(selectionHighlightMirrorAncestor, originalAncestor));
+        SdfPathVector implicitSelectionHighlightChildPaths;
+        for (const auto& originalChildPath : originalChildPaths) {
+            SdfPath explicitSelectionHighlightChildPath = _GetSelectionHighlightMirrorPathFromOriginal(originalChildPath);
+            if (_SelectionHighlightMirrorExists(explicitSelectionHighlightChildPath)) {
+                // There already exists an explicit selection highlight mirror for this child (e.g. point instance prototypes),
+                // so don't create a duplicate implicit one.
```

Comment:
This is just to avoid overdraw, as without the check everything still works, but a highlight could be drawn more than once